forumhelpers/index.html

```diff
@@ -28,6 +28,13 @@ <h3 class="forumhelpers_header">Below is a list of current Forum Helpers, along
 		<br>
 		<p class="forumhelpers_header">There are currently <span id="managersNumber"></span> Managers and <span id="curatorsNumber"></span> Curators in our studio, for a total of <span id="totalNumber"></span> members.</p>
 		<!--<h4 style="margin-left: 10px;"><img src="/resources/coder.png" alt="coder" style="width:30px;height:30px;vertical-align:middle"> Indicates that that person has helped code the website.</h4>-->
+
+		<div class="searchBlock">
+			<p><strong>Search for a Forum Helper</strong></p>
```

Comment:
Could this possibly be a bit bigger?

Reply:
Done, I changed it to a `<h3>` element instead of `<p>`, which increases the font size.

Reply:
Great!